src/main/java/eu/delving/x3ml/engine/Domain.java

```diff
@@ -1,108 +1,171 @@
-//===========================================================================
```

Comment:
Unfortunately, the global reformatting which was done in https://github.com/isl/x3ml/commit/fea4c0be64ca66d928a73fd5a72d2f5ba2485985 prevents a proper line-by-line comparison of the changes.  Comparing file to file is much more time-consuming.